Library/Homebrew/ignorable.rbi

```diff
@@ -0,0 +1,7 @@
+# typed: strict
+
+module Ignorable
+  # This is a workaround to enable `raise` to be aliased
+  # @see https://github.com/sorbet/sorbet/issues/2378#issuecomment-569474238  sig {returns(T.noreturn)}
```

Comment:
Should the `sig` here not be on a separate line?

Reply:
Thanks! This accidentally included files from another branch (for the next typing PR). I'll also squash the code review commits for a cleaner merge.